arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -0,0 +1,1778 @@
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/managerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The encryption settings.",
+            "schema": {
+              "$ref": "#/definitions/EncryptionSettings"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/managers/{managerName}/extendedInformation/vaultExtendedInfo": {
+      "get": {
+        "tags": [
+          "Managers"
+        ],
+        "description": "Returns the extended information of the specified manager name.",
+        "operationId": "Managers_GetExtendedInfo",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/managerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The manager extended information.",
+            "schema": {
+              "$ref": "#/definitions/ManagerExtendedInfo"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "Managers"
+        ],
+        "description": "Creates the extended info of the manager.",
+        "operationId": "Managers_CreateExtendedInfo",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The manager extended information.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagerExtendedInfo"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/managerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The extended information on the manager has been successfully created",
+            "schema": {
+              "$ref": "#/definitions/ManagerExtendedInfo"
```

Comment:
@ravbhatnagar Yes, you are correct, these are proxy resources. 

Also, I saw the following at below path: 
"If any model or its parent is marked with an extension "x-ms-azure-resource" : true, then AutoRest will flatten the Resource-specific properties by one level for that model."

https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#ResourceFlattening

which effectively means, x-ms-azure-resource is used for flattening of properties, but we see that even for proxy resources flattening is happening today, so (a)  what's the actual purpose served by 'x-ms-azure-resource'?
(b)  It's highly likely that tomorrow, this field is used for specific purpose and we would like only 'Manager' to be treated for as azure-resource. Since in all our other Services too we have marked only the Tracked resources as 'x-ms-azure-resource', and hence we would like to follow the same convention here. Please do let us know if that is fine.